docs/Bootloader_Entry.md

```diff
@@ -101,25 +99,29 @@ for information on finding the CAN UUID of your devices.
 
 When klipper receives one of the above bootloader requests:
 
-If Katapult (formerly known as CANBoot) is available, klipper will request that
-Katapult stay active on the next boot, then reset the MCU (therefore entering
-Katapult).
+If [Katapult (formerly known as CANBoot)](Bootloaders.md#katapult-bootloader-lpc176x-stm32-rp2040)
+is available, klipper will request that Katapult stay active on the next boot,
+then reset the MCU (therefore entering Katapult).
 
-If Katapult is not available, klipper will then try to enter a
-platform-specific bootloader, such as STM32's DFU
-mode([see note](#stm32-dfu-warning)).
+If Katapult is not available, klipper will then try to enter a platform-specific
+bootloader, such as STM32's DFU mode ([see note](#stm32-dfu-warning)).
 
 In short, Klipper will reboot to Katapult if installed, then a hardware specific
 bootloader if available.
 
 For details about the specific bootloaders on various platforms see
 [Bootloaders](Bootloaders.md)
 
+---
+
 ## Notes
 
 ### STM32 DFU Warning
 
-Note that on some boards, like the Octopus Pro v1, entering DFU mode can cause
-undesired actions (such as powering the heater while in DFU mode). It is
-recommended to disconnect heaters, and otherwise prevent undesired operations
-when using DFU mode. Consult the documentation for your board for more details.
+!!! danger "STM32 DFU Warning"
```

Comment:
Heh, a line, a header, another header, and a danger..  I think we may be overdoing this a bit.  :-)

Reply:
Year but otherwise without the heading it does not get listed at the TOC.
And iirc the current used mkdocs-material version does not support amonitions without title.